### PR TITLE
Alik search

### PR DESCRIPTION
**Pull request by aliklynnlyle**
@jrmcgarvey @leila-alderman @micronix

### CODE REVIEW

File: placements.sublime-workspace [hunk 1]
```diff
@@ -436,10 +436,29 @@
 	},
 	"expanded_folders":
 	[
-		"/Users/Alicia/Desktop/CTD/placements"
+		"/Users/Alicia/Desktop/CTD/placements",
+		"/Users/Alicia/Desktop/CTD/placements/app",
+		"/Users/Alicia/Desktop/CTD/placements/app/controllers",
+		"/Users/Alicia/Desktop/CTD/placements/app/models",
```
Comment (jrmcgarvey) [inline placements.sublime-workspace]: Alik, I am not sure what this file is, but it looks to be part of your sublime workspace and not something you want to include in your pull request.

Comment (aliklynnlyle) [reply placements.sublime-workspace]: OK, I moved them. They were workspace files.

**Comment by aliklynnlyle** [comment]
@leila-alderman @jrmcgarvey @micronix 
Hoping I resolved these conflicts properly.

**Comment by jrmcgarvey** [comment]
The team decided not to use these changes.

**Pull request closed without merging**